src/operations/indexes.ts

```diff
@@ -1,5 +1,64 @@
-import { OperationBase } from './operation';
 import { indexInformation } from './common_functions';
+import { OperationBase, Aspect, defineAspects } from './operation';
```

Comment:
This file looks like brand new code, but it's just copy pasta from all the other individual index related files. No functional changes were made